dumpling/export/config.go

```diff
@@ -519,6 +526,26 @@ func ParseTableFilter(tablesList, filters []string) (filter.Filter, error) {
 	return filter.NewTablesFilter(tableNames...), nil
 }
 
+func GetConfTables(tablesList []string) (DatabaseTables, error) {
+	dbTables := DatabaseTables{}
+	var (
+		schema, name string
+		tableType    TableType
+		avgRowLength uint64
+	)
+	tableType = TableTypeBase
+	avgRowLength = 0
+	for _, name = range tablesList {
```

Comment:
Suggest setting flag here. Because tablesList is a more precise variable to show whether users has specified `--tables-list`
```suggestion
	conf.specifiedTables = len(tablesList) > 0
	for _, name = range tablesList {
```